### ISSUE TITLE
How to programmatically change svg_raw_resource of SvgImageView

### ISSUE THREAD

**Issue by sonzerolee** [MostafaGazar/CustomShapeImageView#24]
I have a SvgImageView and a Button.
Every time button is clicked, i want to change svg_raw_resource of SvgImageView.
Please help!

**Comment by edurocks** [comment]
Yes i really need this information too. Please help. I want to change the svg dynamically how can i do that?
Thanks
